packages/jmix-react-ui/src/ui/MasterDetail/masterDetailContext.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { createContext, useContext } from "react";
+import { MasterDetailStore } from "./MasterDetailStore";
+
+export const MasterDetailContext = createContext<{
+    masterDetailStore: MasterDetailStore;
+}>({
+    masterDetailStore: new MasterDetailStore(),
+});
+
+export const useMasterDetail = () => {
+    const {masterDetailStore} = useContext(MasterDetailContext);
```

Comment:
Can we avoid creating a global store? Perhaps lift the state to `CarMasterDetails` component? (using `useLocalObservable` if mobx-power is needed)

Reply:
Or `MasterDetail` component.